torch_dipu/csrc_dipu/base/basedef.h

```diff
@@ -11,6 +11,11 @@
 #define DIPU_DEVICE_TYPE_MACRO XPU
 #define DIPU_AUTOGRAD_DEVICE_TYPE_MACRO AutogradXPU
 
+#define DeviceTypeDIPU  at::DeviceType::XPU
+#define AutocastDIPU  AutocastXPU
+
+#define ADD_NS(RAW_OP) at::RAW_OP
```

Comment:
should not be in header IMO, move to where it is used

Reply:
Cancel the macro definition and use it directly